packages/typespec-bump-deps/src/cli/cli.ts

```diff
@@ -27,7 +27,7 @@ async function main() {
   for (const packageJsonPath of packageJsonPaths) {
     const content = await readFile(packageJsonPath);
     const packageJson = JSON.parse(content.toString());
-    const depTypes = ["dependencies", "devDependencies", "peerDependencies"];
+    const depTypes = ["dependencies", "devDependencies", "peerDependencies", "overrides"];
     for (const depType of depTypes) {
       const deps = packageJson[depType];
       if (deps === undefined) continue;
```

Comment:
This is our expected:
1. We bump the version of dependencies in the first package.json.
2. We need to add the overrides section in the second package.json which doesn't have any dependencies. And in that overrides section, there are the bumped dependencies. 

Is that doable?

Reply:
Current script won't.

Should be doable in script. But please confirm that this solution indeed work for .NET, before we actually implement it here.

Reply:
@pshao25 can you share exactly what state of the package.json worked for you. 

I am asking this because the script in its current state only change version of packages already in the package.json, it doesn't add anything. If we want to add new entries we need to make sure we can include the right set of dependencies.

@MaryGao same thing for `rush` could we get the exact value that is working for you

Reply:
I assume they want to add/merge all these libs
https://github.com/Azure/cadl-ranch/blob/781add01b1544fe5df6a428676b0887c879ead85/packages/typespec-bump-deps/src/cli/cli.ts#L5C1-L11

Reply:
It's kind of tech debt, but we have two `pakage.json`s in our repo. The first [one](https://github.com/Azure/autorest.csharp/blob/e8c8b670c684b9680bd8b747f5daedbb138b4759/src/CADL.Extension/Emitter.Csharp/package.json) contains the actual dependecies, which we want to bump the typespec version. The second [one](https://github.com/Azure/autorest.csharp/blob/e8c8b670c684b9680bd8b747f5daedbb138b4759/package.json) adds the first one as dependency, which is the entry we install the package, i.e., we install packages with this package.json. 

Our expectation is: after installation, the root folder has typespec of dev version. The only way I could think of now is: in the second package.json add the overrides section and put all the updated dependecies from the first one into it.

Reply:
Java has 2 package.json too.

You can call it twice, first without the option to typespec project only update the dependencies, second with the option to test project and also add overrides.